pkg/compact/compact.go

```diff
@@ -950,6 +955,17 @@ func (c *BucketCompactor) Compact(ctx context.Context) (rerr error) {
 			return errors.Wrap(err, "build compaction groups")
 		}
 
+		ignoreDirs := []string{}
+		for _, gr := range groups {
+			ignoreDirs = append(ignoreDirs, gr.Key())
+		}
+
+		files, _ := ioutil.ReadDir(c.compactDir)
```

Comment:
I think we need to check err, no?

Reply:
Actually, I have added this while trying to debug e2e tests. It's not needed at all so I have removed it.